src/containers/DefaultLayout/DefaultLayout.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import * as router from 'react-router-dom';
+import { Container } from 'reactstrap';
+import { removeAuthenticatedState, isAuthUserType } from '../../services/Auth'
+
+import {
+  AppFooter,
+  AppHeader,
+  AppSidebar,
+  AppSidebarMinimizer,
+  AppBreadcrumb2 as AppBreadcrumb,
+  AppSidebarNav2 as AppSidebarNav,
+} from '@coreui/react';
+// sidebar nav config
+import { customer, agent, admin} from '../../_nav';
+// import navigation from '../../_nav';
+// routes config
+import routes from '../../routeList';
+import DefaultFooter from './DefaultFooter'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['DefaultFooter' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)